packages/twenty-server/src/engine/metadata-modules/field-metadata/validators/is-field-metadata-default-value.validator.ts

```diff
@@ -14,13 +14,17 @@ import {
   FieldMetadataType,
 } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
 import { validateDefaultValueForType } from 'src/engine/metadata-modules/field-metadata/utils/validate-default-value-for-type.util';
+import { LoggerService } from 'src/engine/integrations/logger/logger.service';
 
 @Injectable()
 @ValidatorConstraint({ name: 'isFieldMetadataDefaultValue', async: true })
 export class IsFieldMetadataDefaultValue
```

Comment:
In this class you can do something like that, if you want the error sent to the front-end to be more specific:

```typescript
@Injectable()
@ValidatorConstraint({ name: 'isFieldMetadataDefaultValue', async: true })
export class IsFieldMetadataDefaultValue
  implements ValidatorConstraintInterface
{
  private validationErrors: string[] = [];

  constructor(private readonly fieldMetadataService: FieldMetadataService) {}

  async validate(
    value: FieldMetadataDefaultValue,
    args: ValidationArguments,
  ): Promise<boolean> {
    // Try to extract type value from the object
    let type: FieldMetadataType | null = args.object['type'];

    if (!type) {
      // Extract id value from the instance, should happen only when updating
      const id: string | undefined = args.instance?.['id'];

      if (!id) {
        return false;
      }

      let fieldMetadata: FieldMetadataEntity;

      try {
        fieldMetadata = await this.fieldMetadataService.findOneOrFail(id);
      } catch {
        return false;
      }

      type = fieldMetadata.type;
    }

    const { isValid, errors } = validateDefaultValueForType(type, value);

    if (errors.length > 0) {
      this.validationErrors = errors.map((error) => error.toString());
    }

    return isValid;
  }

  defaultMessage(): string {
    if (this.validationErrors.length > 0) {
      return this.validationErrors.join(', ');
    }

    return 'FieldMetadataDefaultValue is not valid';
  }
}
```